src/components/views/rooms/EntityTile.tsx

```diff
@@ -82,13 +95,16 @@ class EntityTile extends React.Component {
         showPresence: true,
     };
 
-    state = {
-        hover: false,
-    };
+    constructor(props: IProps) {
+        super(props);
+
+        this.state = {
+            hover: false,
+        };
+    }
 
-    shouldComponentUpdate(nextProps, nextState) {
+    shouldComponentUpdate(nextProps: IProps, nextState: IState) {
         if (this.state.hover !== nextState.hover) return true;
-        return this.props.shouldComponentUpdate(nextProps, nextState);
     }
```

Comment:
`shouldComponentUpdate` defaultProp used to be `() => true` so we either need a `return true` here but I think we can just forgo the shouldComponentUpdate and switch this to a PureComponent